### ISSUE TITLE
Uncaught TypeError: editor.transact is not a function

### ISSUE THREAD

**Issue by soraxas** [tsuyoshiwada/sequential-number#11]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.21.2 ia32
**Electron**: 1.6.15
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [sequential-number](https://github.com/tsuyoshiwada/sequential-number) package 0.5.0

### Stack Trace

Uncaught TypeError: editor.transact is not a function

```
At C:\Users\Oscar\.atom\packages\sequential-number\lib\sequential-number.coffee:50

TypeError: editor.transact is not a function
    at Object.exec (/packages/sequential-number/lib/sequential-number.coffee:50:14)
    at /packages/sequential-number/lib/sequential-number.coffee:11:34
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.19.3/resources/app/node_modules/scrollbar-style/node_modules/event-kit/lib/emitter.js:25:14)
    at SequentialNumberView.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.19.3/resources/app/node_modules/scrollbar-style/node_modules/event-kit/lib/emitter.js:125:28)
    at SequentialNumberView.module.exports.SequentialNumberView.handleKeyup (/packages/sequential-number/lib/sequential-number-view.coffee:61:8)
```

### Commands

```
     -3:00.8.0 intentions:highlight (input.hidden-input)
     -3:00.1.0 sequential-number:open (input.hidden-input)
     -2:48.5.0 command-palette:toggle (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -2:44.8.0 core:confirm (input.hidden-input)
     -2:44.8.0 settings-view:open (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -0:23.7.0 intentions:highlight (input.hidden-input)
  6x -0:23.3.0 editor:duplicate-lines (input.hidden-input)
 10x -0:22.5.0 core:move-up (input.hidden-input)
     -0:21.1.0 intentions:highlight (input.hidden-input)
 19x -0:21 find-and-replace:select-next (input.hidden-input)
     -0:19 intentions:highlight (input.hidden-input)
     -0:18.7.0 sequential-number:open (input.hidden-input)
     -0:16.4.0 core:confirm (input.hidden-input)
     -0:00.9.0 command-palette:toggle (input.hidden-input)
```

### Non-Core Packages

```
activate-power-mode 1.4.0 
aligner-javascript 1.1.0 
atom-alignment 0.13.0 
atom-beautify 0.29.23 
atom-latex 0.8.2 
atom-material-syntax 1.0.2 
atom-material-ui 1.3.10 
atom-python-run 0.7.3 
autocomplete-python 1.8.63 
busy-signal 1.4.1 
git-control 0.9.0 
intentions 1.1.2 
language-abaqus 0.2.0 
language-latex 1.0.0 
language-tex 0.4.0 
latex 0.49.0 
linter 2.1.4 
linter-spell 0.13.2 
linter-spell-latex 0.9.2 
linter-ui-default 1.2.3 
minimap 4.28.1 
minimap-git-diff 4.3.1 
minimap-hide 0.3.0 
python-indent 1.0.2 
python-tools 0.6.9 
sequential-number 0.5.0 
split-diff 1.4.0 
sync-settings 0.8.1 
```

**Comment by wadackel** [comment]
Hi @soraxas Thank you for report :+1:

Can you tell me what kind of input to the Input Panel will cause an error?
Thanks.

**Comment by soraxas** [comment]
Hi @tsuyoshiwada sorry for not including the procedures that cause the issue before.

It wasn't any special input, was just **1** , **1 + 1**, or any other pure numbers. That was when I only installed the package for the first time and tried it out. After a few restart (restarting only Atom, not the OS) it still throws the same error. I was in a hurry into finishing a project before, so I just went ahead and used another similar package for incrementing multiple numbers.

But today after seeing your comment I reinstalled it again and it works perfectly without any error as of yet. I suppose there was something wrong when it was installed initially. Sorry, I couldn't reproduce it again for you, it's all good now. I will close the issue.
